app/components/podcasts.hbs

```diff
@@ -59,7 +60,10 @@
         {{else}}
           {{t "podcasts.no_result"}}
         {{/each}}
-        <Pagination @meta={{result.meta}} @route="home.podcasts" />
+        <Pagination
+          @totalPages={{result.meta.total_pages}}
+          @page={{@page}}
```

Comment:
Here `@page` is derived from the query string. 
Seems hackier than just using the `result.meta.page`, but I still can't figure out why this section is not re-rendering like I expect. -_-